app/App/HomeController.php

```diff
@@ -18,49 +18,49 @@
 class HomeController extends Controller
 {
     /**
-     * Display the homepage.
-     */
+    * Display the homepage.
+    */
     public function index(Request $request, ActivityQueries $activities)
     {
         $activity = $activities->latest(10);
         $draftPages = [];
-
+        
         if ($this->isSignedIn()) {
             $draftPages = Page::visible()
-                ->where('draft', '=', true)
-                ->where('created_by', '=', user()->id)
-                ->orderBy('updated_at', 'desc')
-                ->with('book')
-                ->take(6)
-                ->get();
+            ->where('draft', '=', true)
+            ->where('created_by', '=', user()->id)
```

Comment:
```suggestion
            ->where('draft', true)
            ->where('created_by', user()->id)
```

Default operator is the `=` sign, this change improves readability